coding/html.md

```diff
@@ -9,3 +9,7 @@ General rules for all HTML documents are:
  - Tags and attributes should be lowercase (`<p class="intro">` not `<P CLASS="intro"`)
  - Use double quotes for attribute values
  - Close all elements - either a closeing tag (`<p>...</p>`) or self-closing (`<img src="jeff.jpg" />`)
+
+## Images
+  - Use alt attributes on img elements
+  - Use null alt text and no title attribute on img elements for images that Assistive Technology should ignore
```

Comment:
Could you clarify this a bit? What is null alt text? Do we have a reference we could link to about this point?